lib/invisible_captcha.rb

```diff
@@ -7,7 +7,12 @@
 
 module InvisibleCaptcha
   class << self
-    attr_accessor :sentence_for_humans, :error_message, :honeypots, :visual_honeypots
+    attr_accessor :sentence_for_humans,
+                  :error_message,
+                  :honeypots,
+                  :threshold,
+                  :threshold_error_message,
```

Comment:
Does make sense to rename it to `:timestamp_threshold` and `:timestamp_error_message`?